app/_includes/lts-changes.html

```diff
@@ -0,0 +1,28 @@
+{% for c in site.data.tables.breaking_changes_lts.categories %}
+<h3 id="{{ c.name | slugify }}">{{ c.name }}</h3>
+<p>{{ c.desc | markdownify }}</p>
+<table>
+  <thead>
+    <th>Change</th>
+    <th>Category</th>
+    <th>Action required</th>
```

Comment:
Is it possible to remove the ones that do not require an action from the table? Just thinking that if people don't need to perform an action, they probably don't care what the new feature is?

Reply:
Unfortunately no, but I can remove the column. This is one of the categories of items that, for the LTS upgrade, they wanted to be sure users are notified - the idea is that they shouldn't need to look through the changelog and parse what's important for the LTS, since it's all covered here.